ocamltest/dune

```diff
@@ -41,8 +41,9 @@
  (flags (:standard -nostdlib))
  (libraries ocamlcommon stdlib)
  (modules (:standard \ options main))
+ (modules_without_implementation ocamltest_unix)
```

Comment:
I don’t think this can be right - it needs a `select` stanza to get ocamltest_unix.ml from one of the two template files

Reply:
I am not sure I follow, trunk doesn't have ocamltest_unix.ml, unless you're meaning the `select` stanza would pick from *_dummy.ml and *_real.ml as the template file? In which case I would be happy to revise.

Reply:
That's correct - `ocamltest_unix.ml` is `ocamltest_unix_dummy.ml` when the Unix library is not available and `ocamltest_unix_real.ml` otherwise (see [here](https://github.com/ocaml/ocaml/blob/trunk/ocamltest/Makefile#L36), [here](https://github.com/ocaml/ocaml/blob/trunk/ocamltest/Makefile#L42), and [here](https://github.com/ocaml/ocaml/blob/trunk/ocamltest/Makefile#L253-L255) in `ocamltest/Makefile`). I would expect `ocamltest` to fail to link as given here.

I think it would be:

```
(select ocamltest_unix.ml from (unix -> ocamltest_unix_real.ml) (-> ocamltest_unix_dummy.ml)
```
in `libraries`. That said, I don't think the Dune build "supports" building without unix, so you could also just have a rule to copy `ocamltest_unix_real.ml` to `ocamltest_unix.ml`

Reply:
I made the proposed changes. Follow up on a new review.